R/matrix_form.R

```diff
@@ -40,7 +40,26 @@ mform_handle_newlines <- function(matform) {
   line_grouping <- unique(line_grouping)
 
   # nlines detects if there is a newline character
-  row_nlines <- apply(strmat, 1, function(x) max(vapply(x, nlines, 1L), 1L))
+  # colwidths = NULL, max_width = NULL, fontspec = NULL
+  # because we don't care about wrapping here we're counting lines
+  # TODO probably better if we had a nlines_nowrap fun to be more explicit
+
+  row_nlines <- apply(
+    strmat,
+    1,
+    function(x) {
+      max(
+        vapply(x,
+          nlines,
+          colwidths = NULL,
```

Comment:
nlines accepts colwidths because the TableRow method  for it (which live in rtables) needs them to do wrapping to determine numbers of lines. I'm saying here explicitly that no colwidths are being used because we're not operating on an entire row

Reply:
understood, added to description